watchdog/main.go

```diff
@@ -250,6 +250,34 @@ func makeRequestHandler(config *WatchdogConfig) func(http.ResponseWriter, *http.
 	}
 }
 
+func isLockFileExists() bool {
+	path := filepath.Join(os.TempDir(), ".lock")
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		return false
+	}
+	return true
+}
+
+func health() func(http.ResponseWriter, *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		switch r.Method {
+		case "GET":
```

Comment:
@stefanprodan does Kubernetes care about the method for a health endpoint? should it accept anything and still respond the same way?

Reply:
It should be GET with 200 or 500 response code

Reply:
But is the validation necessary?